core/image.cpp

```diff
@@ -2473,7 +2473,8 @@ void Image::fix_alpha_edges() {
 
 					if (rp[3] < alpha_threshold)
 						continue;
-
+                    
+                    closest_dist=dist;
```

Comment:
This line is still indented with spaces. Please use tabs.

Reply:
Can you check it again. In Xcode IDE i have 5 tabs here

Reply:
Still the same. If it's changed, the comment will fold automatically.